src/materials/MeshPhysicalMaterial.js

```diff
@@ -23,6 +23,8 @@ function MeshPhysicalMaterial( parameters ) {
 	this.clearCoat = 0.0;
 	this.clearCoatRoughness = 0.0;
 
+	this.sheen = 0.0;
```

Comment:
It will be necessary to enhance the docs and TypeScript declaration files with this parameter. However, you can do this with a different PR after this one gets merged.

Reply:
Done